### DISCUSSION TITLE
[Art competition] Public Key Cryptography

### DISCUSSION THREAD

**Discussion by 727yubin** [prosyslab-classroom/cs348-information-security#417] [Art competition]
![craiyon_123114_A_heroic_painting_of_a_medieval_knight_fighting_valiantly_in_battle__mask_off__with_emphasis_on_a_scar_on_his_face_](https://github.com/prosyslab-classroom/cs348-information-security/assets/14364385/5801b2bf-0724-49e7-8a48-7f713178c10e)

## Short Description
For me, public key cryptography means "A heroic painting of a medieval knight fighting valiantly in battle, with emphasis on his scar on his face, with no face mask."

## Long Description (up to 3 sentences)
In public key cryptography, each party has a public key and a corresponding private key. The public key is known to everyone, but the corresponding private key should be kept secret. This is similar to a scar. A scar is easily visible, and everyone can see it. However, the story behind the scar, and the complete background, is known only to the person him/herself.

Note: this was generated by [Craiyon](https://www.craiyon.com/), not DALLE (due to no free credits provided anymore).

**Comment by KihongHeo** [comment]
Good. It would be nice if your metaphor also describes other properties of public-key crypto.
